assets/js/editor-components/incompatible-extension-notice/modal.tsx

```diff
@@ -12,7 +12,7 @@ export const ModalContent = ( {
 		return (
 			<p>
 				{ __(
-					'If you continue, the cart block will be replaced with the classic experience powered by shortcode. This means that you may lose customizations and updates you did to the cart block.',
+					'If you continue, the cart block will be replaced with the classic experience powered by shortcodes. This means that you may lose customizations that you did to the cart block.',
```

Comment:
```suggestion
					'If you continue, the cart block will be replaced with the classic experience powered by shortcodes. This means that you may lose customizations that you made to the cart block.',
```

Doesn't this read better?